pkg/didcomm/transport/http/inbound.go

```diff
@@ -33,9 +35,9 @@ func NewInboundHandler(prov transport.Provider) (http.Handler, error) {
 		return nil, errors.New("creation of inbound handler failed")
 	}
 
-	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+	return cors.Default().Handler(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
maybe split this into a couple lines :).

Reply:
[Done](https://github.com/hyperledger/aries-framework-go/compare/701dca0d75e2995e951d53faa06cef20c2c16ff7..b394e22f2fdb5682cd144096a72968273a6843ef#diff-18bf71ef9d572ae715312cab65be25eeR38-R42)